project-management/src/main/java/life/qbic/projectmanagement/application/rawdata/RawDataLookup.java

```diff
@@ -0,0 +1,28 @@
+package life.qbic.projectmanagement.application.rawdata;
+
+import java.util.Collection;
+import java.util.List;
+import life.qbic.projectmanagement.application.SortOrder;
+import life.qbic.projectmanagement.domain.model.measurement.MeasurementId;
+import life.qbic.projectmanagement.domain.model.rawdata.RawData;
+
+public interface RawDataLookup {
+
+  /**
+   * Queries {@link RawData} with a provided offset and limit that supports
+   * pagination.
+   *
+   * @param measurementIds  the list of {@link MeasurementId} for which the {@link RawData}
+   *                   should be fetched
+   * @param filter     the results fields will be checked for the value within this filter
+   * @param offset     the offset for the search result to start
+   * @param limit      the maximum number of results that should be returned
+   * @param sortOrders the ordering to sort by
+   * @return the results in the provided range
+   */
+  List<RawData> queryRawDataByMeasurementIds(String filter,
```

Comment:
The ordering of the returned list will be irrelevant since it solely depends on the sortorder(at least as far as i understood it). However I changed the provided parameter to be a list to avoid confusion good point!